### ISSUE TITLE
FPS limit on virtual camera

### ISSUE THREAD

**Issue by derhellehof** [ludgerh/cam-ai#88]
The new virtual camera feature works, but the FPS limit does'nt do anything.

**Comment by ludgerh** [comment]
You are right, there is room for improvement. I removed the brake by purpose because I think if you are processing files you might want to do this as fast as possible. Also, some people might want to just watch and for these the right speed would be important. I think that two additional flags would be good:
a) Process in original speed (Yes/No)
b) Process everything: Not to omit frames if the feed is too fast. this requires a powerful PC...
